app/models/topic.rb

```diff
@@ -0,0 +1,14 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Topic < ApplicationRecord
+  validates :name, presence: true
```

Comment:
Eventually you could also add a `uniqueness: true` validation, so that you don't have more than 1 topic with the same `name`.

And remember to add model specs for validations.